doc/source/user/quickstart.rst

```diff
@@ -1346,12 +1358,15 @@ and then use it as::
     array([[[15, 14, 16, 18, 13],
             [12, 11, 13, 15, 10],
             [11, 10, 12, 14,  9]],
+    <BLANKLINE>
            [[16, 15, 17, 19, 14],
             [13, 12, 14, 16, 11],
             [12, 11, 13, 15, 10]],
+    <BLANKLINE>
            [[17, 16, 18, 20, 15],
             [14, 13, 15, 17, 12],
             [13, 12, 14, 16, 11]],
+    <BLANKLINE>
```

Comment:
@mattip, did we delivery choose to avoid using BLANKLINE?

Reply:
The recent file absolute_beginners.rst uses BLANKLINE, for instance. But refguide_check.py does not care, it is true. I was led into putting them by pytest.

Reply:
We tried to work around `<BLANKLINE>` but there are places we couldn't avoid it. I don't remember the details. My conclusion when messing with it was that we should move to [astropy's pytest-doctestplus](https://github.com/astropy/pytest-doctestplus/) rather than enhance refguide-check.

Reply:
It seems you must either mash the blocks together without a blank line between them or use the `<BLANKLINE>` directive to indicate that formatting leaves a blankline there. The first is a bit misleading, I think I prefer the way it is now.

```

Reply:
Should I attempt to resolve the remaining failures:

```
ERROR:  failed checking doc/source/reference/arrays.nditer.rst
        failed checking doc/source/user/tutorial-svd.rst
        failed checking doc/source/user/c-info.python-as-glue.rst
        failed checking doc/source/f2py/getting-started.rst
```
Or should I test drive the astropy tool?

Reply:
I think we should push forward with the current tool. Once we have cleaned up the last failures, we can add the check to CI, then the transition to the new tool can proceed in a controlled way.